tests/pipelines/test_pipelines_summarization.py

```diff
@@ -55,7 +51,18 @@ def run_pipeline_test(self, summarizer, _):
         )
         self.assertEqual(outputs, [{"summary_text": ANY(str)}])
 
-        if not isinstance(model.config, (SwitchTransformersConfig, T5Config, LongT5Config, LEDConfig)):
+        model_can_handle_longer_seq = [
+            "SwitchTransformersConfig",
+            "T5Config",
+            "LongT5Config",
+            "LEDConfig",
+            # tokenizers that get small `model_max_length`
+            "PegasusXConfig",  # `PegasusXModelTester` sets `max_position_embeddings=30`
+            "FSMTConfig",  # `FSMTModelTester` sets `max_position_embeddings=20`
+            "M2M100Config",  # `M2M100ModelTester` sets `max_position_embeddings=20`
+            "ProphetNetConfig",  # `ProphetNetModelTester` sets `max_position_embeddings=30`
+        ]
```

Comment:
In  some model testers, we have `max_position_embeddings`  being 20, 30, but also 512 in other ones (`BertModelTester, GPT2ModelTester` etc). The tokenizers' `max_model_length` is set to `max_position_embeddings` (if existing) during tiny model creation,  so for those with smaller values, the long input here will be truncated, and we don't get expected exception.